doc/classes/InputEventShortcut.xml

```diff
@@ -4,7 +4,7 @@
 		Represents a triggered keyboard [Shortcut].
 	</brief_description>
 	<description>
-		InputEventShortcut is a special event that can be received in [method Node._unhandled_key_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
+		InputEventShortcut is a special event that can be received in [method Node._shortcut_input] and [method Node._unhandled_key_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```

Comment:
Adding `_shortcut_input` is correct, but the preexisting text isn't.

So you should either name the function, that is primarily intended for shortcuts:
```suggestion
		InputEventShortcut is a special event that can be received in [method Node._shortcut_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```

or include all functions, that shortcuts can be received in:
```suggestion
		InputEventShortcut is a special event that can be received in [method Node._input], [method Node._shortcut_input] and [method Node._unhandled_input]. It is typically sent by the editor's Command Palette to trigger actions, but can also be sent manually using [method Viewport.push_input].
```